app/classifier/tasks/drawing/summary.jsx

```diff
@@ -19,7 +41,17 @@ class DrawingSummary extends React.Component {
     translation: {}
   }
 
+  state = {
+    expanded: false
+  }
+
+  toggleExpanded() {
+    const { expanded } = this.state;
+    this.setState({ expanded: !expanded });
```

Comment:
This could be instead:

`this.setState((prevState) => { return { expanded: !prevState.expanded } })`

Reply:
Good catch!

Reply:
I've updated the `setState` call, rebased and restaged.

Reply:
I don't see the update to the `setState` call here or on my local copy after pulling down again.

Reply:
It would probably help if I actually saved changes before committing. Should be good now.